### DISCUSSION TITLE
Azure Static Web App Password Protection not working

### DISCUSSION THREAD

**Discussion by williamrousseau** [unoplatform/uno#12204] [Q&A]
I want to host the template Uno Platform WebAssembly app on Azure by following the [Hosting Uno Platform WebAssembly apps on Azure Static Web Apps](https://platform.uno/blog/hosting-uno-platform-webassembly-apps-on-azure-static-web-apps/) steps. This works great and the Web app is accessible. </br>

But, I have to [Configure the Password protection](https://learn.microsoft.com/en-us/azure/static-web-apps/password-protection) to have a minimal security implemented and making sure that my app can't be seen by anyone. **The password make the application crash**. Indeed, after that the password has been entered, the application will not load and stays on the splash screen.

![image](https://user-images.githubusercontent.com/43794005/236882730-a874f9b4-b334-4fb1-a238-9a22a43ca768.png)

Why is it crashing specifically when adding a password protection?

**Comment by jeromelaban** [comment]
Thanks for the question! Indeed, it's a quite curious error, probably the result of failing queries. I don't beleive we've made explicit support for this type of authentication using azure static apps.

You can take a look at the network tab in the debugger tools to determine what is happening here, and see which queries are failing.
